app/assets/stylesheets/govuk_publishing_components/components/_table.scss

```diff
@@ -125,3 +125,39 @@ $table-row-even-background-colour: govuk-colour("light-grey");
     border: 0;
   }
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .govuk-table--sortable {
+    outline: none;
+
+    * {
+      color: $govuk-print-text-colour !important;
+      background-color: initial !important;
+    }
+
+    .govuk-table__header,
+    .govuk-table__cell {
+      border-bottom: 1px solid govuk-colour("mid-grey");
+    }
+
+    .govuk-table__header {
+      font-weight: 700;
+      border-right: 0;
+    }
+
+    .app-table__sort-link {
+      padding: 0 !important;
+
+      &::before,
+      &::after {
+        display: none !important;
+      }
+    }
+  }
+
+  .js-gem-c-table__filter {
+    display: none !important;
```

Comment:
Doesn't seem like `important` is needed? Doesn't look like this class has any other declarations set anywhere.

Reply:
I've removed this declaration in favour of the `govuk-!-display-none-print` class on the filter element in the view.